doc/ext/convert-svg-to-pdf.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+"""
+    Converts SVG images to PDF using chrome in case the builder does not
+    support SVG images natively (e.g. LaTeX).
+
+"""
+import subprocess
+
+from sphinx.errors import ExtensionError
+from sphinx.locale import __
+from sphinx.transforms.post_transforms.images import ImageConverter
+from sphinx.util import logging
+from errno import ENOENT, EPIPE, EINVAL
+import os
+from sys import platform as pf
+import tempfile
+
+if False:
+    # For type annotation
+    from typing import Any, Dict  # NOQA
+    from sphinx.application import Sphinx  # NOQA
+
+
+logger = logging.getLogger(__name__)
+
+
+class Converter(ImageConverter):
+    conversion_rules = [
+        ('image/svg+xml', 'application/pdf'),
+    ]
+
+    def is_available(self):
+        # type: () -> bool
+        """Confirms if converter is available or not."""
+        return True
+
+    def chrome_command(self):
+         # type: () -> str
+        if pf == 'win32':
+            return os.path.join(os.environ['PROGRAMW6432'],
+             "Google\Chrome\Application\chrome.exe")
+        elif pf == 'darwin':
+            return "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"
+        elif pf == 'linux':
+            return "google-chrome"
+        else:
+            logger.warning("Not able to find suitable chrome command for this os")
+
+    def convert(self, _from, _to):
+        # type: (unicode, unicode) -> bool
+        """Converts the image from SVG to PDF using chrome."""
+
+        HTML = "<html ><head ><style >body {margin: 0; }</style ><script >function init() {const element = document.getElementById('targetsvg');const positionInfo = element.getBoundingClientRect();const height = positionInfo.height;const width = positionInfo.width;const style = document.createElement('style');style.innerHTML = \`@page {margin: 0; size: \${width}px \${height}px}\`;document.head.appendChild(style); }window.onload = init;</script ></head><body><img id=\"targetsvg\" src=\"%s\"></body></html>" % (_from)
+        chrome = self.chrome_command()
+        try:
+            temp = tempfile.NamedTemporaryFile(suffix=".html", mode='w', delete=False)
+            with open(temp.name, 'w+') as f:
+                f.write(HTML)
+                f.seek(0)
+            if pf == 'win32':
+                args = '"%s" --headless --disable-gpu --disable-software-rasterizer --print-to-pdf="%s" "%s"' % (chrome, _to, temp.name)
+            else:
+                args = '%s --headless --disable-gpu --disable-software-rasterizer --print-to-pdf=%s %s' % (chrome, _to, temp.name)
+            logger.info(args)
+            logger.debug('Invoking %r ...', args)
+            p = subprocess.Popen(
+                args, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
```

Comment:
You need to pass a list of arguments to subprocess, like `['google-chrome', '--headless', '--disable-gpu', ...]`. 

It also is usually simpler to use `subprocess.run` instead of `Popen`.